render-wasm/src/shapes/renderable.rs

```diff
@@ -317,13 +319,15 @@ pub fn draw_image_stroke_in_container(
     canvas.draw_image_rect(image, None, dest_rect, &image_paint);
 
     // Clear outer stroke for paths if necessary. When adding an outer stroke we need to empty the stroke added too in the inner area.
-    if let (Kind::Path(p), StrokeKind::OuterStroke) = (kind, &stroke.kind) {
-        let mut path = p.to_skia_path();
-        path.transform(path_transform.unwrap());
-        let mut clear_paint = skia::Paint::default();
-        clear_paint.set_blend_mode(skia::BlendMode::Clear);
-        clear_paint.set_anti_alias(true);
-        canvas.draw_path(&path, &clear_paint);
+    if let Kind::Path(p) = kind {
+        if stroke.kind(p.is_open()) == StrokeKind::OuterStroke {
```

Comment:
Not sure about this condition. Is there any other possibilities that `stroke.kind()` returns something different than `StrokeKind::CenterStroke` in the future?

If not, I would keep the original `let` guard as it was (`(Kind::Path(p), StrokeKind::OuterStroke)`) , and just check for `!p.is_open` here.

Reply:
Sorry but I don't get it..could you pase some example code?

stroke.kind() can return something different than StrokeKind::CenterStroke right now, it depends on the internal kind attribute and if the path is open or not. I don't know how to keep the original let comparing StrokeKind::OuterStroke with stroke.kind(p.is_open())